sdk/iot/doc/how_to_iot_hub_samples_windows.md

```diff
@@ -0,0 +1,408 @@
+# How to setup and run Azure SDK for Embedded C IoT Hub Samples on Microsoft Windows
+
+_Last edited on June 3rd, 2020._
+
+This is a step-by-step documentation of how to start from scratch and get the Azure SDK for Embedded C IoT Hub Samples running on Microsoft Windows. 
+
+Pre-requisites:
+- [Having created an Azure account](https://github.com/ewertons/azure-sdk-for-c/wiki/How-to-create-an-Azure-account)
+- [Having created an Azure IoT Hub](https://github.com/ewertons/azure-sdk-for-c/wiki/How-to-create-an-Azure-IoT-Hub)
+- Microsoft Visual Studio installed in the local machine.
+
+What is covered:
+- Downloading and building the Azure SDK for Embedded C suite
+- Configuring and running the IoT Hub client samples.
+
+_Run on Microsoft Windows 10.0.18363.836._
+
+01. Install git
+
+  Get the installer from the official git [page](https://git-scm.com/download/win)
+
+02. Install the latest cmake
+
+Check the latest version available on https://cmake.org/download/.
+
+After installing, check if cmake works correctly: 
+
+```shell
+C:\>cmake --version
+cmake version 3.15.19101501-MSVC_2
+
+CMake suite maintained and supported by Kitware (kitware.com/cmake).
+
+C:\> 
+```
+
+03. Install Paho using vcpkg
+ 
+The Azure IoT SDK for C uses Eclipse Paho for C installed via vcpkg (for the cmake integration).
```

Comment:
The Azure IoT SDK for C samples uses Eclipse Paho....

Reply:
Actually that is correct. They do have different versions for different languages... But maybe it's redundant. Ok, I'll remove.